src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -204,7 +195,7 @@ private Card getCard(int cardType, StepParameters stepParameters) {
     }
 
     private Card createJobStartedCard() {
-        String jobName = decodeURIComponent(run.getParent().getDisplayName());
+        String jobName = run.getParent().getDisplayName();
```

Comment:
I found - also based on my experiences from another plugin - that encoding is not needed. Maybe it was when the ``name`` was used, but now we have ``displayName`` which works fine.